support/Pipelines/Bbh/Ringdown.yaml

```diff
@@ -46,6 +46,7 @@ DomainCreator:
 Evolution:
   InitialTime: 0.0
   InitialTimeStep: 0.0002
+  MinimumTimeStep: 1e-6
```

Comment:
I think for the ringdown we want this to also be `1e-7`

Reply:
Oops.  Missed the pipeline files.  Fixed.